app/views/amends/AmendExportedPlasticPackagingView.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components.{heading, paragraphBody, bulletList, LinkButton}
+
+@this(
+    layout: templates.Layout,
+    govukButton: GovukButton,
+    heading: heading,
+    paragraphBody: paragraphBody,
+    bulletList: bulletList,
+    linkButton: LinkButton
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("amendExportedPlasticPackaging.title"), Some(messages("return.title.tail")))) {
+
+    @heading("amendExportedPlasticPackaging.heading")
+
+    @paragraphBody(messages("amendExportedPlasticPackaging.paragraph.1"))
+    @paragraphBody(messages("amendExportedPlasticPackaging.paragraph.2"))
+    @bulletList(elements = Seq(
+            Html(messages("amendExportedPlasticPackaging.bulletPoint.1")),
+            Html(messages("amendExportedPlasticPackaging.bulletPoint.2"))
+        )
+    )
+
+    @paragraphBody(messages("amendExportedPlasticPackaging.paragraph.3"))
+
+    @linkButton(
+        messageKey = "site.button.continue",
+        linkUrl = "/amend-exported-weight"
```

Comment:
reverse routes!

Reply:
`@controllers.routes.AmendExportedWeightController.onPageLoad`

Reply:
just put something there as navigation not done yet.